src/test/java/edu/kis/powp/jobs2d/events/SelectRotateCurrentCommandOptionListener.java

```diff
@@ -0,0 +1,21 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.command.ICompoundCommand;
+import edu.kis.powp.jobs2d.command.visitor.TransformationsVisitor;
+import edu.kis.powp.jobs2d.drivers.transformation.Rotate;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectRotateCurrentCommandOptionListener implements ActionListener {
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        ICompoundCommand command = (ICompoundCommand) CommandsFeature.getDriverCommandManager().getCurrentCommand();
```

Comment:
Casting is not necessary with visitor!
Please try to remove dependency to CommandsFeature (it would help with future refactoring)